cfme/tests/webui/test_rest.py

```diff
@@ -0,0 +1,61 @@
+import fauxfactory
+import pytest
+
+from cfme.infrastructure.provider import InfraProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.rest import delete_resources_from_collection
+from cfme.utils.rest import delete_resources_from_detail
+
+
+@pytest.fixture(params=[True, False], ids=["global", "local"])
+def search_filter_obj(appliance, request):
+    filter_name = fauxfactory.gen_string("alphanumeric", 10)
+    filter_value = fauxfactory.gen_string("alphanumeric", 10)
+    param_filter = "Infrastructure Provider : Name"
+
+    view = navigate_to(InfraProvider, "All")
```

Comment:
instead of importing InfraProvider,  I think it would be better to navigate via `appliance.collections.infra_providers`

Reply:
I pushed a commit for this after testing locally.